kafka-lib_0_10/src/main/scala/Consumer.scala

```diff
@@ -6,6 +6,16 @@ import scala.util.Try
 trait Consumer[T] {
 
   def kafkaConsumer: KafkaConsumer[String, String]
+  
+  def processBatch(
```

Comment:
Should't be breaking as it's just adding new methods? I'd rather leave out any impl as it's up to the implementor to choose if they want to support it or not